internal/core/application/types.go

```diff
@@ -16,6 +16,12 @@ import (
 	"github.com/vulpemventures/go-elements/transaction"
 )
 
+type WalletInfo struct {
```

Comment:
little comment about what this `Wallet` refers to (ie. not the wallet account)

Maybe I would call it `HDWalletInfo` or `VaultInfo` ?